docs/changes/coordinates/11676.feature.rst

```diff
@@ -0,0 +1,3 @@
+Adds new ``HADec`` built-in frame with transformations to/from ``ICRS`` and ``CIRS`` 
+along with basic tests. This frame complements ``AltAz`` to give observed coordinates 
```

Comment:
Unless the user is supposed to run these tests, there is no need to mention them.
```suggestion
Adds new ``HADec`` built-in frame with transformations to/from ``ICRS`` and ``CIRS``.
This frame complements ``AltAz`` to give observed coordinates 
```

Reply:
I clicked on Commit suggestion. Does that actually make the change that you suggested?

Reply:
I guess it did.